kolibri/core/assets/src/views/sync/FacilityNameAndSyncStatus/index.vue

```diff
@@ -111,6 +112,12 @@
         }
         return this.$formatRelative(datetime, { now: this.now });
       },
+      manageSync() {
+        this.$router.push({
+          name: SyncPageNames.MANAGE_SYNC_SCHEDULE,
+          params: { facilityId: this.facility.id },
```

Comment:
This is still going to cause issues, because this component is being used in the device plugin and the facility plugin. The facility plugin uses `facility_id` as a route parameter, and the device plugin is currently using `facilityId` as a route parameter. As long as they use a different parameter, this is going to cause issues.

For the sake of moving forward, I think it might be simplest, just to standardize the new device plugin routes you have created to use `facility_id` as the route parameter for facility, so that we can set this consistently.

Ideally, this is why this shouldn't be handled internally to the component, so that this standalone component doesn't need to 'know' about the details of the route implementations in the different plugins, but we can open follow up issues to handle this as technical debt.